src/main/java/creativei/helper/ResponseHelper.java

```diff
@@ -36,28 +36,28 @@ public static InquiryVo getInquiryResponseById(Inquiry inquiry) {
     }
 
     public static List<InquiryVo> getAllInquiryResponse(List<Inquiry> inquiries) {
-        List<InquiryVo> inquiryVos=new ArrayList<>();
-        for(Inquiry inquiry:inquiries){
-            InquiryVo inquiryVo=new InquiryVo(inquiry);
+        List<InquiryVo> inquiryVos = new ArrayList<>();
+        for (Inquiry inquiry : inquiries) {
+            InquiryVo inquiryVo = new InquiryVo(inquiry);
             inquiryVos.add(inquiryVo);
         }
         return inquiryVos;
     }
 
     public static List<InquiryVo> getInquiryResponseByStatus(List<Inquiry> inquiries) {
-        List<InquiryVo> inquiryVos=new ArrayList<>();
-        for(Inquiry inquiry:inquiries){
-            InquiryVo inquiryVo=new InquiryVo(inquiry);
+        List<InquiryVo> inquiryVos = new ArrayList<>();
+        for (Inquiry inquiry : inquiries) {
+            InquiryVo inquiryVo = new InquiryVo(inquiry);
             inquiryVos.add(inquiryVo);
         }
         return inquiryVos;
     }
 
 
-    public static List<InquiryVo> getUnattendedInquiryResponse(List<Inquiry> inquiries){
-        List<InquiryVo> inquiryVos=new ArrayList<>();
-        for(Inquiry inquiry:inquiries){
-            InquiryVo inquiryVo=new InquiryVo(inquiry);
+    public static List<InquiryVo> getUnattendedInquiryResponse(List<Inquiry> inquiries) {
+        List<InquiryVo> inquiryVos = new ArrayList<>();
+        for (Inquiry inquiry : inquiries) {
```

Comment:
Remove unused reference. Directly add new object to the list

Reply:
removed in commit 1a6703f